e2e/cypress/support/command.js

```diff
@@ -24,76 +24,103 @@
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
 function verify_element_exists() {
-    cy.get('td').eq(4).then(($td) => {
-        if ($td.text() == 'Finished') {
-            cy.wrap($td.text()).should('eq', 'Finished');
-        } else {
-            cy.wait(10000);
-            cy.reload(true);
-            verify_element_exists();
-        }  
-    })
+  cy.get("td")
+    .eq(4)
+    .then(($td) => {
+      if ($td.text() == "Finished") {
+        cy.wrap($td.text()).should("eq", "Finished");
+      } else {
+        cy.wait(5000);
+        cy.reload(true);
+        verify_element_exists();
+      }
+    });
```

Comment:
I didn't realize we don't have a limit here; this recursive function can go on forever.
We might want to implement it like [this](https://github.com/GSA/catalog.data.gov/blob/main/e2e/cypress/support/command.js#L1-L15), with the function call like [this](https://github.com/GSA/catalog.data.gov/blob/1174275d74f321b7ebc171bdb029f05ec9b1b756/e2e/cypress/support/command.js#L241).